docs/source/configuration/entity-caching.mdx

```diff
@@ -128,9 +128,157 @@ This entry contains an object with the `all` field to affect all subgraph reques
       "data": "key2"
     }
 }
+```
+
+### Invalidation
+
+Entity caching provides multiple ways to invalidate data:
+- an invalidation endpoint exposed by the router, that can receive invalidation requests from any authorized service out of band. This can be used by the subgraph to asynchronously trigger invalidation events, or by the platform operator to maintain the caching levels
+- invalidation requests can be sent through subgraph response extensions, allowing the subgraph to invalidate cached data right after a mutation
```

Comment:
```suggestion
- **Invalidation endpoint** - the router exposes an invalidation that can receive invalidation requests from any authorized service out-of-band. A subgraph can use this endpoint to asynchronously trigger invalidation events, or a platform operator can use it to maintain the caching levels.
- **Subgraph response extensions** - you can send invalidation requests via subgraph response extensions, allowing a subgraph to invalidate cached data right after a mutation.
```

Reply:
Minor correction to Edward's copy

- **Invalidation endpoint** - the router exposes an invalidation **endpoint** that can receive invalidation requests from any authorized service out-of-band. A subgraph can use this endpoint to asynchronously trigger invalidation events, or a platform operator can use it to maintain the caching levels.